amplify/src/main/java/com/github/stkent/amplify/tracking/managers/ApplicationChecksManager.java

```diff
@@ -74,16 +74,36 @@ public boolean shouldAllowFeedbackPrompt() {
             final long installTime = applicationInfoProvider.getInstallTime();
 
             //noinspection ConstantConditions
-            result = result && installTimeEventCheck.shouldAllowFeedbackPrompt(installTime);
+            boolean installResult = installTimeEventCheck.shouldAllowFeedbackPrompt(installTime);
+
+            if (!installResult) {
+                logger.d("Blocking prompt based on install time check");
+            }
+
+            result = result && installResult;
         }
 
         if (lastUpdateTimeTimeEventCheck != null) {
```

Comment:
lastUpdateTimeTimeEventCheck -> lastUpdateTimeEventCheck

![burn-it](https://cloud.githubusercontent.com/assets/6463980/12201938/0621ff28-b5f5-11e5-8843-06113edf8025.gif)